File: customized-checkout/webhooks/listener/php/listen_payment.php
<?php
require_once "mercadopago.php";

$mp = new MP("ACCESS_TOKEN");

$json_event = file_get_contents('php://input', true);
$event = json_decode($json_event);

if (!isset($event->type, $event->data) || !ctype_digit($event->data->id)) {
	http_response_code(400);
	return;
}

if ($event->type == 'payment'){
    $payment_info = $mp->get('/v1/payments/'.$event->data->id);

    if ($payment_info["status"] == 200) {
        print_r($payment_info["response"]);
    }
}
?>

File: customized-checkout/webhooks/listener/ruby/listen_payment.rb
require 'rubygems'
require 'rack'
require 'mercadopago'

class WebHook
 	def call(env)
		mp = MercadoPago.new('ACCESS_TOKEN')

		req = Rack::Request.new(env)

		event = JSON.parse (env['rack.input'].gets)

		if (event['type'] == 'payment')
			payment_info = mp.get("/v1/payments/"+event['data']['id']);

			return [200, {'Content-Type' => 'application/json'}, [payment_info.to_json]]
		else
			return [400, {'Content-Type' => 'text/plain'}, ['No payment event']]
		end
	end
end

Rack::Handler::WEBrick.run(WebHook.new, :Port => 80)